storybookjs/storybook
# Addons: Fix Viewport resetting shortcut key not working

Issue: https://github.com/storybookjs/storybook/issues/17519

## What I did
implementing `event.code` in `eventToShortcut` method to support `alt (option) + <key>` on macos

## How to test
1) Run `yarn task --task dev --template react-vite/default-ts`
2) Navigate to any story in the app
3) Change the viewport by pressing `V` key
4) Ensure that `alt + V` reset viewport

## Demo

https://www.loom.com/share/f172b62597c140a2a2365d493d88bfc6
